pageserver/src/layered_repository.rs

```diff
@@ -110,6 +110,18 @@ lazy_static! {
     .expect("failed to define a metric");
 }
 
+/// The name of the metadata file pageserver creates per timeline.
+pub const METADATA_FILE_NAME: &str = "metadata";
+
+/// All files and data, relevant to a certain pageserver timeline,
+/// found locally in its working directory.
+#[derive(Debug)]
+struct LoadedLayerMap {
+    pub layers: Vec<PathBuf>,
+    /// Latest Lsn that corresponds to the contents written to disk.
+    pub disk_consistent_lsn: Option<Lsn>,
```

Comment:
When will this be false? If we can't find the metadata, should we just reject loading the layer map all together?

Reply:
Looks like after I've dealt with double metadata loading issue, this struct is not needed, we just return the file paths and don't care about the Lsn.